src/store/migrations/14_rsk_derivation_path.js

```diff
@@ -0,0 +1,75 @@
+import { getDerivationPath } from '@/utils/derivationPath'
+import { Networks } from '@/utils/networks'
+import { accountCreator, getNextAccountColor } from '@/utils/accounts'
+import { ChainId, chains } from '@liquality/cryptoassets'
+
+export const rskDerivationPath = {
+  version: 14,
+  migrate: async (state) => {
+    const hasAccounts = Object.keys(state.accounts || {}).length > 0
+
+    if (!hasAccounts) {
+      return {
+        ...state
+      }
+    }
+
+    const _accounts = {}
+
+    for (const walletId in state.accounts) {
+      _accounts[walletId] = {}
+
+      for (const network of Networks) {
+        const accounts = state.accounts[walletId][network]
+        const updatedAccounts = []
+        for (const account of accounts) {
+          const derivationPath = getDerivationPath(
+            account.chain,
+            network,
+            account.index,
+            account.type
+          )
+          const updatedAccount = {
+            ...account,
+            alias: '',
+            addresses: [],
+            balances: {},
+            derivationPath
+          }
+          updatedAccounts.push(updatedAccount)
+
+          if (account.chain === ChainId.Rootstock) {
```

Comment:
Will this add 2 accounts for ledger too? 
We need to decide what to do there, do you know other people using ledger, which path do they usually get? for example on metamask

Right now, we didn't migrate users to the new derivation path for ledger.

Reply:
Agree, fixed